build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Nit: add some comments to explain the flow so that it's clear for everyone, then looks ready :)